lib/streamlit/elements/selectbox.py

```diff
@@ -135,6 +136,8 @@ def selectbox(
             An optional tuple of args to pass to the callback.
         kwargs : dict
             An optional dict of kwargs to pass to the callback.
+        placeholder : str
+            An optional string to display when option is not selected.
```

Comment:
nit: can very slightly tweak this to be "An optional string to display when an option is not selected."

Reply:
https://github.com/streamlit/streamlit/pull/6913/commits/f2108be5a3de8c64bf808323b8e5d993c391d8c2
How about the same modification as st.multiselect?